src/litserve/server.py

```diff
@@ -472,3 +472,100 @@ def setup_auth(self):
         if LIT_SERVER_API_KEY:
             return api_key_auth
         return no_auth
+
+
+@asynccontextmanager
+async def manage_lifespan(app: FastAPI, servers: List[LitServer]):
```

Comment:
this is really nice. the function name however is not that intuitive. what do you think about `multi_server_lifespan` or `merged_lifespan`?

Reply:
`multi_server_lifespan` sounds good to me.